Cargo.toml

```diff
@@ -16,9 +16,7 @@ no-default-features = true
 
 [dependencies]
 rustls = { version = "0.23", default-features = false, features = ["std"] }
-sha2 = "0.10"
 windows-sys = { version = "0.59", features = ["Win32_Foundation", "Win32_Security_Cryptography"] }
-aws-lc-rs = { version = "1", optional = true }
```

Comment:
Dmitry, OK to remove these aws-lc- dependencies?

Reply:
Yes I think it can be removed, also the aws-lc-bindgen feature should be removed as well. The reason I had it is that the ealier versions of aws-lc-rs didn't have the pre-built bindings for some platforms, so the compilation of rustls was failing.